api/api.go

```diff
@@ -349,7 +354,34 @@ func (w *ApiWrapper) SignJwt(ctx echo.Context) error {
 	return ctx.String(http.StatusOK, sig)
 }
 
-func (w *ApiWrapper) Verify(ctx echo.Context) error {
+// SignTLSCertificate creates a certificate based on a PEM encoded public key. It uses the Vendor CA to sign.
+// the resulting certificate is valid for 4 days.
```

Comment:
"for 4 days" << this depends on how it's spec-ed & configured and an implementation detail of the service, so no need to mention it here.